flow/connectors/mysql/validate.go

```diff
@@ -139,6 +139,8 @@ func (c *MySqlConnector) CheckBinlogSettings(ctx context.Context, requireRowMeta
 }
 
 func (c *MySqlConnector) ValidateMirrorSource(ctx context.Context, cfg *protos.FlowConnectionConfigs) error {
+	return nil
```

Comment:
is this TODO?

Reply:
I screwed up & leaked code from testing earlier